apps/server/src/modules/task/controller/dto/submission.response.ts

```diff
@@ -0,0 +1,39 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+
+export class SubmissionStatusResponse {
+	constructor({ id, submitters, isSubmitted, grade, isGraded, submittingCourseGroupName }: SubmissionStatusResponse) {
+		this.id = id;
+		this.submitters = submitters;
+		this.isSubmitted = isSubmitted;
+		this.grade = grade;
+		this.isGraded = isGraded;
+		this.submittingCourseGroupName = submittingCourseGroupName;
+	}
+
+	@ApiProperty()
+	id: string;
+
+	@ApiProperty()
+	submitters: string[];
```

Comment:
@CeEv Die Submitters sind aktuell nur UserIds. In der UI zeigen wir auch die Namen der User an. Aus diesem Grund haben wir uns gefragt, ob wir die Namen über diese Schnittstelle ausliefern wollen. Es bräcuhte dafür dann wahrscheinlich sowas wie nen UserService und da sind wir uns unsicher, ob wir das in diesem Ticket wirklich bauen wollen.

Reply:
Lassen wir unpopulated